tests/autoyast/installation.pm

```diff
@@ -303,6 +304,9 @@ sub run {
         elsif (match_has_tag 'expired-gpg-key') {
             send_key 'alt-y';
         }
+        elsif (match_has_tag 'registration-codes') {
+            send_key 'alt-n';
```

Comment:
this screen https://openqa.suse.de/tests/9427059#step/installation/6
will not be here when the NVIDIA repo will exist:
https://openqa.suse.de/tests/8753030#step/installation/5
We should process all this 'workaround' flow above in the same `elseif` I mean and you don't need to add it to `@needles` it is expected that one thing is after the other in this case.